Lamouresparus/Pantha.ng
# Pan 2

## Tracked by PAN-2
## This PR adds
The registration screens with both active and inactive states
## Considerations and Implementations
The registrations Screens
## How to test

1. Build and run app
2. click on register
3. click request access
4. fill the fields to see the states 

Bonus: password validation added on the confirm password field

## Test(s) Added
None
![Screenshot_20210424-172219_Pantha ng](https://user-images.githubusercontent.com/15072297/115966033-bc81e580-a523-11eb-8a08-4af7b9a99417.png)
![Screenshot_20210424-172224_Pantha ng](https://user-images.githubusercontent.com/15072297/115966034-bdb31280-a523-11eb-9244-6a91d9c694ad.png)
![Screenshot_20210424-172229_Pantha ng](https://user-images.githubusercontent.com/15072297/115966036-be4ba900-a523-11eb-9b40-3213584921ba.png)
![Screenshot_20210424-172239_Pantha ng](https://user-images.githubusercontent.com/15072297/115966037-bf7cd600-a523-11eb-9436-72ff00952f59.png)
![Screenshot_20210424-172254_Pantha ng](https://user-images.githubusercontent.com/15072297/115966039-c0156c80-a523-11eb-9ca0-fc7921ecd893.png)
![Screenshot_20210424-172339_Pantha ng](https://user-images.githubusercontent.com/15072297/115966040-c1469980-a523-11eb-8bbd-aba86b06df25.png)
![Screenshot_20210424-172351_Pantha ng](https://user-images.githubusercontent.com/15072297/115966041-c1df3000-a523-11eb-9217-ecb685c1c19a.png)
![Screenshot_20210424-172355_Pantha ng](https://user-images.githubusercontent.com/15072297/115966043-c3105d00-a523-11eb-9e2d-81689ce15ec6.png)
